File: Wikipedia/Code/WMFWelcomeLanguageTableViewCell.m
#import "WMFWelcomeLanguageTableViewCell.h"

@interface WMFWelcomeLanguageTableViewCell ()

@property (strong, nonatomic) IBOutlet UILabel *languageNameLabel;

@end

@implementation WMFWelcomeLanguageTableViewCell

- (void)setLanguageName:(NSString *)languageName {
    _languageName = languageName;
    self.languageNameLabel.text = languageName;
}

@end

File: Wikipedia/Code/WMFWelcomeLanguageTableViewCell.swift

class WMFWelcomeLanguageTableViewCell: WMFCustomDeleteButtonTableViewCell {
    @IBOutlet var languageNameLabel:UILabel!
    var languageName: String? {
        set (newLanguageName){
            languageNameLabel.text = newLanguageName
        }
        get {
            return languageNameLabel.text
        }
    }
    override func awakeFromNib() {
        super.awakeFromNib()
        languageNameLabel.wmf_configureSubviewsForDynamicType()
    }
}
